specification/caas-published-cohort-definitions.yaml

```diff
@@ -27,6 +27,188 @@ info:
     To be completed.
     ## Onboarding
     To be completed.
+
+    ### Sandbox testing
```

Comment:
The postman requests don't have any tests - Could you please add some?
![image](https://github.com/NHSDigital/caas-published-cohort-definitions/assets/85311517/c7a60aff-d079-4411-bb24-d07e7d9de637)

Reply:
✅ tests added

Reply:
<img width="898" alt="image" src="https://github.com/NHSDigital/caas-published-cohort-definitions/assets/120651954/bb5cf4a2-c457-4a1e-a4d3-5846948bf1c1">